homeassistant/components/snapcast/media_player.py

```diff
@@ -192,6 +194,8 @@ def __init__(
 
         self._device = device
         self._attr_unique_id = self.get_unique_id(host_id, device.identifier)
+        self._attr_media_content_type = MediaType.MUSIC
+        self._attr_device_class = MediaPlayerDeviceClass.SPEAKER
```

Comment:
This can be set outside of the constructor

Reply:
Done.